conductr_cli/bndl_utils.py

```diff
@@ -35,14 +35,8 @@ def detect_format_dir(dir):
             os.path.isfile(os.path.join(dir, 'repositories')) and \
             os.path.isfile(os.path.join(dir, 'manifest.json')):
         return 'docker'
-    elif \
-            os.path.isfile(os.path.join(dir, 'runtime-config.sh')):
-        return 'bundle'
-    elif \
-            os.path.isfile(os.path.join(dir, 'bundle.conf')):
-        return 'bundle'
     else:
-        return None
+        return 'bundle'
```

Comment:
I'm not sure we need to do this - given that we don't autodetect configuration, isn't returning None if there is no `bundle.conf` appropriate?